WordPress/Classes/Services/Stories/StoryEditor.swift

```diff
@@ -112,16 +128,60 @@ class StoryEditor: CameraController {
                  tagCollection: nil,
                  saveDirectory: saveDirectory)
 
-        cameraHandler = CameraHandler(created: { [weak self] _ in
+        cameraHandler = CameraHandler(created: { [weak self] media in
+            self?.poster = StoryPoster(context: post.blog.managedObjectContext ?? ContextManager.shared.mainContext, mediaFiles: mediaFiles)
+            let postMedia: [StoryPoster.MediaItem] = media.compactMap { result in
+                switch result {
+                case .success(let item):
+                    guard let item = item else { return nil }
+                    return StoryPoster.MediaItem(url: item.output, size: item.size, archive: item.archive, original: item.unmodified)
+                case .failure:
+                    return nil
+                }
+            }
+
+            guard let self = self else { return }
+            let uploads: (String, [Media])? = self.poster?.upload(mediaItems: postMedia, post: post as! Post, completion: { post in
```

Comment:
⚠️ Force unwrap ⚠️ 😄

Reply:
Whoops! Thought I caught all of these. Thanks!